apps/server/src/modules/tool/school-external-tool/service/school-external-tool.service.ts

```diff
@@ -57,20 +57,33 @@ export class SchoolExternalToolService {
 		tool: SchoolExternalTool,
 		externalTool: ExternalTool
 	): Promise<ToolConfigurationStatus> {
+		const configurationStatus: ToolConfigurationStatus = new ToolConfigurationStatus({
+			isDisabled: false,
+			isOutdatedOnScopeContext: false,
+			isOutdatedOnScopeSchool: true,
+		});
+
+		const latestConfigurationStatus: ToolConfigurationStatus = new ToolConfigurationStatus({
```

Comment:
Do you really need two instances ? I could imagine a static function or class which you could use here